sdcm/nemesis.py

```diff
@@ -2338,10 +2338,11 @@ def disrupt_hot_reloading_internode_certificate(self):
         if not self.cluster.params.get('server_encrypt', None):
             raise UnsupportedNemesis('Server Encryption is not enabled, hence skipping')
 
-        @retrying(allowed_exceptions=LogContentNotFound)
+        @retrying(n=6, allowed_exceptions=(LogContentNotFound, ))
         def check_ssl_reload_log(target_node, file_path, since_time):
-            msg = target_node.remoter.run(f'journalctl -u scylla-server --since="{since_time}" | '
-                                          f'grep Reload | grep {file_path}', ignore_status=True)
+            reloaded_msg = 'messaging_service - Reloaded {' + ssl_files_location + '}'
```

Comment:
you can use fstring, then it will be like:
```suggestion
            reloaded_msg = 'messaging_service - Reloaded {{{ssl_files_location}}}'
```

Reply:
https://media1.giphy.com/media/3o7btXkbsV26U95Uly/200.gif?cid=6104955ed501bf644b62aa241bfa8dae6c019ac3c84f7290&rid=200.gif